src/styles/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
You have set the grid-template-columns property to repeat(1, 200px), which means it will create 1 column of 200px. This is fine for mobile first design, but you need to ensure that this property is properly overridden in the media queries.